travel_platform/izi_travel_client.py

```diff
@@ -0,0 +1,306 @@
+                mus["image_url"] = image_url
+                mus["language"] = museum.get("language")
+                mus["map"] = museum.get("map")
+                mus["location"] = museum.get("location")
+                museums.append(mus)
+            return museums
+        return museums
+
+    def get_city_tours(self, city):
+        """
+        Get list of city tours.
+        Args:
+            city (str): City name.
+        Returns:
+            list: List of tours.
+        """
+        city_uuid = self.get_city_uuid(city)
+        url = '{base_url}cities/{city}/children'.format(
+            base_url=self.api_url, city=city_uuid)
+        params = self._prepare_params()
+        response = requests.get(url, params, headers=self.headers)
+        self._check_response_status(response)
+        tours = []
+        city_data = response.json()
+        if city_data:
+            for obj in city_data:
+                if obj.get("type") == "tour":
+                    tours.append(obj)
+            return tours
+        return tours
+
+    def get_museum_detail_with_audio(self, museum_uuid):
+        """
+        Get details about museum with audio, reviews, description.
+        Args:
+            museum_uuid (str): Museum uuid.
+        Returns:
+            dict: Details about certain museum.
+        """
+        url = "{base_url}mtgobjects/{museum}".format(
+            base_url=self.api_url, museum=museum_uuid)
+        prepared_params = {"includes": "download,city",
+                           "except": "publisher,children"}
+        params = self._prepare_params(**prepared_params)
+        response = requests.get(url, params, headers=self.headers)
+        self._check_response_status(response)
+        museum_data = response.json()
+        detail_museum_info = {}
+        if museum_data:
+            content_provider = museum_data[0].get("content_provider")
+            detail_museum_info["name"] = content_provider.get("name")
+            content_provider_uuid = content_provider.get("uuid")
+            content = museum_data[0]["content"][0]
+            audio_uuid = content["audio"][0]["uuid"]
+            audio_url = self._prepare_audio_url(content_provider_uuid,
+                                                audio_uuid)
+            detail_museum_info["audio"] = audio_url
+            description = content["desc"]
+            detail_museum_info["description"] = description
+            detail_museum_info["address"] = museum_data[0].get("contacts")
+            images = content["images"]
+            detail_museum_info["reviews"] = museum_data[0].get("reviews")
+            detail_museum_info["images"] = museum_data[0].get("images")
+            for image in images:
+                im_url = self._prepare_image_url(content_provider_uuid,
+                                                 image.get("uuid"))
+                detail_museum_info["images"].append(im_url)
```

Comment:
It looks like it's a `list` type, but we need some additional validation here, because I've got an error:
```
Traceback (most recent call last):
  File "izi_travel_client.py", line 298, in <module>
    print client.get_museum_detail_with_audio(mus_uuid)
  File "izi_travel_client.py", line 197, in get_museum_detail_with_audio
    detail_museum_info["images"].append(im_url)
AttributeError: 'NoneType' object has no attribute 'append'
```

So, I guess, it will be great to improve this a little bit.
As a simple solution we may replace:
`detail_museum_info["images"] = museum_data[0].get("images")`
with
`detail_museum_info["images"] = museum_data[0].get("images", [])`

P.S. I've added this hotfix by myself.